modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/snapshot/DataApplicationSnapshotAcceptanceManager.scala

```diff
@@ -181,13 +179,16 @@ object DataApplicationSnapshotAcceptanceManager {
           validatedBlocks.traverse(service.serializeBlock)
         )
 
+        feeTransactions <- OptionT.liftF {
+          Async[F].delay {
```

Comment:
does `getFeeTransactions` make a side effect?

Reply:
I don't think so, maybe it can be pure